### PR TITLE
[fix](ut)support run be-ut on release mode

### PR DESCRIPTION
**Pull request by zclllyybb**
# Proposed changes

## Problem summary

Fixed improper usage. So now be ut could be run on release mode.
btw, split be build type environment variable to be/be-ut.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

File: be/test/util/threadpool_test.cpp [hunk 1]
```diff
@@ -328,15 +328,9 @@ TEST_F(ThreadPoolTest, TestZeroQueueSize) {
...
-            "_ZNSt3__1L8__invokeIRNS_6__bindIMN5doris10ThreadPoolEFvvEJPS3_EEEJEEEDTclscT_fp_"
-            "spscT0_fp0_EEOS9_DpOSA_|_ZNSt3__18__invokeB6v15007IRNS_6__"
-            "bindIMN5doris10ThreadPoolEFvvEJPS3_EEEJEEEDTclclsr3stdE7declvalIT_"
-            "EEspclsr3stdE7declvalIT0_EEEEOS9_DpOSA_";
+    const char* death_msg = "ThreadPool";
 #else
-    const char* death_msg =
-            "_ZNSt5_BindIFMN5doris10ThreadPoolEFvvEPS1_EE6__callIvJEJLm0EEEET_OSt5tupleIJDpT0_"
-            "EESt12_Index_tupleIJXspT1_EEE";
+    const char* death_msg = "ThreadPool";
```
Comment (adonis0147) [inline be/test/util/threadpool_test.cpp:333]: I don't think this modification is suitable. If we run BE UT whose build type isn't release, we can check the stack frames more accurately.

**Review by adonis0147 — DISMISSED** [review]

File: be/test/util/threadpool_test.cpp (hunk 1 above)
Comment (zclllyybb) [reply be/test/util/threadpool_test.cpp]: Fixed. but AFAIC, this testcase is simple and clear enough to assert that the only possibility that deadlock occurs is what we expect(in ThreadPool). actually the msg origin is just to check the "call from ThreadPool" too.

**Comment by zclllyybb** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.05 seconds
 stream load tsv:          441 seconds loaded 74807831229 Bytes, about 161 MB/s
 stream load json:         24 seconds loaded 2358488459 Bytes, about 93 MB/s
 stream load orc:          71 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230327081827_clickbench_pr_121233.html

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**